integration-tests/src/test/resources/integration-test.anchor-config.yaml

```diff
@@ -1,170 +1,271 @@
+                  "max_amount": 1000000
+              },
+              "send": {
+                  "fee_fixed": 0,
+                  "fee_percent": 0,
+                  "min_amount": 1,
+                  "max_amount": 1000000
+              },
+              "sep31" : {
+                  "quotes_supported": true,
+                  "quotes_required": false,
+                  "sep12": {
+                      "sender": {
+                          "types": {
+                              "sep31-sender": {
+                                  "description": "U.S. citizens limited to sending payments of less than $10,000 in value"
+                              },
+                              "sep31-large-sender": {
+                                  "description": "U.S. citizens that do not have sending limits"
+                              },
+                              "sep31-foreign-sender": {
+                                  "description": "non-U.S. citizens sending payments of less than $10,000 in value"
+                              }
+                          }
+                      },
+                      "receiver": {
+                          "types": {
+                              "sep31-receiver": {
+                                  "description": "U.S. citizens receiving JPY"
+                              },
+                              "sep31-foreign-receiver": {
+                                  "description": "non-U.S. citizens receiving JPY"
+                              }
+                          }
+                      }
+                  },
+                  "fields": {
+                      "transaction": {
+                          "receiver_routing_number": {
+                              "description": "routing number of the destination bank account"
+                          },
+                          "receiver_account_number": {
+                              "description": "bank account number of the destination"
+                          },
+                          "type": {
+                              "description": "type of deposit to make",
+                              "choices": [
+                                  "SEPA",
+                                  "SWIFT"
+                              ]
+                          }
+                      }
+                  }
+              },
+              "sep38": {
+                  "exchangeable_assets": [
+                      "iso4217:USD"
+                  ]
+              },
+              "sep24_enabled": false,
+              "sep31_enabled": true,
+              "sep38_enabled": true
+          },
+          {
+              "schema": "iso4217",
+              "code": "USD",
+              "deposit" : {
+                  "enabled": true,
+                  "fee_minimum": 0,
+                  "fee_percent": 0,
+                  "min_amount": 0,
+                  "max_amount": 10000
+              },
```

Comment:
For USD --> USDC deposit, I think we can delegate to the [Fee API](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/docs/03%20-%20Implementing%20the%20Anchor%20Server/Communication/Callbacks%20API.yml#L82) and remove the fee and/or min/max info here.

This also mirrors the `sep-24/info` [endpoint](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#info). 

For exchanges, the exchange API will take care of it. 

@marcelosalloum and I actually discussed this. We kept this here because it kind of mirrors the `/info` endpoint. Because it is not clear, we decided to keep it the same way.

TBH, I think we have the `/fee` endpoint. We should probably delegate the request to the `/fee` endpoint.

Reply:
What is the exchange API?

And I agree, every attribute other than `enabled` is optional SEP-24, SEP-6, & SEP-31's `GET /info` response. We can simplify configuration simply by defaulting to that information not being specified.

Reply:
[#568](https://github.com/stellar/java-stellar-anchor-sdk/issues/568) is created for further discussion and implementation.